store/store.go

```diff
@@ -52,6 +55,18 @@ func (cfg *Config) Check() error {
 		return fmt.Errorf("redis password is set, but endpoint is not")
 	}
 
+	// NOTE: we take the MaxBlobLengthBytes from verify package as it is done in memstore
```

Comment:
what about the case where the user specifies both a private key and a remote signer config?

Reply:
in this case we will create a client which will use web3signer
Are you suggesting to fail check and explicitly tell a user to pass params for only one of them?

Reply:
I added a check
if there will be explicitly allowed to have multiple different signers at the same time it will be another story